Drv/LinuxI2cDriver/LinuxI2cDriverComponentImpl.cpp

```diff
@@ -80,8 +80,8 @@ namespace Drv {
 
 #if DEBUG_PRINT
       printf("I2c addr: 0x%02X\n",addr);
-      for (U32 byte = 0; byte < serBuffer.getsize(); byte++) {
-    	  printf("0x%02X ",((U8*)serBuffer.getdata())[byte]);
+      for (U32 byte = 0; byte < serBuffer.getSize(); byte++) {
+    	  printf("0x%02X ",((U8*)serBuffer.getData())[byte]);
```

Comment:
I think this cast is wrong as well. U8 should go in `0x%02X`, not (U8*), which is a pointer. I don't think you need the cast.

Reply:
I think this is fixed, we index into a U8* to get a U8.